packages/web-components/src/components/modal/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,81 @@
+import { Props, Description } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import ExpressiveModal from './ExpressiveModal';
+
+# Expressive modal
+
+> The `<dds-modal>` component is based on
+> [Carbon's Modal component](https://custom-elements.carbondesignsystem.com/?path=/story/components-modal--default-story)
+> with slight styling updates to increase readability and reduce strain.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+##### JS
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/modal/modal';
+import '@carbon/ibmdotcom-web-components/es/components/modal/modal-header';
+import '@carbon/ibmdotcom-web-components/es/components/modal/modal-heading';
+import '@carbon/ibmdotcom-web-components/es/components/modal/modal-close-button';
+import '@carbon/ibmdotcom-web-components/es/components/modal/modal-body';
+import '@carbon/ibmdotcom-web-components/es/components/modal/modal-footer';
+```
+
+##### HTML
+
+```html
+<dds-modal open>
```

Comment:
@asudoh are there instructions on how to trigger the modal to open and close?

Reply:
Good question - Added the doc.